docs/installing-packages.md

```diff
@@ -0,0 +1,203 @@
+└── rpms
+    ├── gpg-keys
+    │   └── git.key
+    └── git-2.35.3-150300.10.33.1.x86_64.rpm
+```
+
+EIB config file `packages` configuration:
+```yaml
+operatingSystem:
+  packages:
+    additionalRepos: <your-reg-code>
+```
+
+### Installing unsigned packages
+By default EIB does GPG validation for every **additional repository** and **side-loaded RPM**. If you wish to use unsigned additional repositories and/or unsinged RPMs you must add the `noGPGCheck: true` property to EIB's `packages` configuration, like so:
+```yaml
+operatingSystem:
+  packages:
+    noGPGCheck: true
+```
+By providing this configuration, **all** GPG validation will be **disabled**, allowing you to use non-signed pacakges.
+
+> **_NOTE:_** This property is intended for development purposes only. For production use-cases we encourage users to always use EIB's GPG validation.
+
+## Package installation workflow
+The package installation workflow can be separated in three logical parts:
+1. *Running the EIB container* - how to run the EIB container so that the **RPM resolution** has the needed permissions
+2. *Building the EIB image* - what happens during the **RPM resolution** logic of EIB's image build
+3. *Booting the EIB image* - how are the packages actually installed once the EIB image is booted for the first time
+
+### Running the EIB container
+![image](./images/rpm-eib-container-run.png)
+
+The package installation workflow begins with the user configuring packages and/or stand-alone RPMs that will be installed when the EIB image is booted. On how to to do a correct configuration, see [Configure packages for installation](#configure-packages-for-installation).
+
+After the desired configuration has been made, the user runs the EIB container with the [`--privileged`](https://docs.podman.io/en/latest/markdown/podman-run.1.html#privileged) option, ensuring that EIB has the needed permissions to successfully run a Podman instance within its container. This is a crutial prerequisite for building a working EIB image with package installation configured (more on this in the next section). 
+
+An example of the command can be seen below:
+```shell
+podman run --rm --privileged -it \
+-v $IMAGE_DIR:/eib eib:dev /bin/eib \
+-config-file $CONFIG_FILE.yaml \
+-config-dir /eib \
+-build-dir /eib/_build
+```
+
+> **_NOTE:_** Depending on the `cgroupVersion` that Podman operates with, you might also need to run the command with `root` permissions. This is the case for `cgroupVersion: v1`. In this version, non-root usage of the `--privileged` option is not supported. For `cgroupVersion: v2`, non-root usage is supported. 
+>
+>In order to check the `cgroupVersion` that Podman operates with, run the following command:
+>```shell
+>podman info | grep cgroupVersion
+>```
+
+Once the EIB container has been successfully executed, it parses all the user provided configuration and begins the **RPM resolution** process. 
+
+### Building the EIB image
```

Comment:
This feels like it should be its own doc covering the design/implementation. The top half of this is much more user based. We don't really have a huge precedent for this yet, but could you split this part out into a subdirectory under docs called `design` so we have an example in place for when (read: if) we get to documenting things more thoroughly? I imagine the k8s and user workload work also needs some sort of design doc in there.